scripts.d/support/install-setup-ng

```diff
@@ -48,7 +82,7 @@ else
 fi
 
 # In auto-mode, do nothing more
-[ $AUTO = 1 ] && exit 0
+[ $AUTO = 1 ] && template_conversion $PWD && exit 0
```

Comment:
You cannot do that as if template_conversion fails it won't exit.

It should be:
`
[ $AUTO = 1 ] && ( template-conversion $PWD ; exit 0 )`